Sat Sep  3 10:24:17 2022
Linux-5.15.0-46-generic-x86_64-with-glibc2.31
Ren'Py 8.0.3.22083102
 

Bootstrap to the start of init.init took 0.04s
Early init took 0.00s
Loader init took 0.00s
Loading error handling took 0.03s
Loading script took 0.65s
Loading save slot metadata. took 0.01s
Loading persistent took 0.00s
Faled to initialize steam: OSError('/home/renan/Downloads/renpy-8.0.3-sdk/lib/py3-linux-x86_64/libsteam_api.so: cannot open shared object file: No such file or directory')
Running init code took 0.06s
Loading analysis data took 0.00s
Analyze and compile ATL took 0.00s
Index archives took 0.00s
Dump and make backups. took 0.03s
Cleaning cache took 0.00s
Making clean stores took 0.00s
Initial gc. took 0.02s
DPI scale factor: 1.000000
Creating interface object took 0.00s
Cleaning stores took 0.00s
Init translation took 0.03s
Build styles took 0.00s
Load screen analysis took 0.00s
Analyze screens took 0.02s
Save screen analysis took 0.02s
Prepare screens took 0.11s
Save pyanalysis. took 0.01s
Save bytecode. took 0.03s
Running _start took 0.00s
Performance test:
Interface start took 0.11s

Initializing gl2 renderer:
primary display bounds: (0, 0, 1280, 720)
swap interval: 1 frames
Windowed mode.
Vendor: "b'Intel'"
Renderer: b'Mesa Intel(R) UHD Graphics 620 (WHL GT2)'
Version: b'4.6 (Compatibility Profile) Mesa 21.2.6'
Display Info: None
Screen sizes: virtual=(1280, 720) physical=(1099, 618) drawable=(1099, 618)
Could not open 'cache/shaders.txt':
Maximum texture size: 4096x4096
Screen sizes: virtual=(1280, 720) physical=(1280, 656) drawable=(1280, 656)
Could not open 'cache/shaders.txt':
Maximum texture size: 4096x4096
Cleaning stores took 0.00s
Init translation took 0.00s
Build styles took 0.02s
Load screen analysis took 0.05s
Analyze screens took 0.00s
Prepare screens took 0.00s
Running _start took 0.00s
